src/screens/FeedScreen/Header.js

```diff
@@ -378,16 +377,18 @@ const _renderProfileNormal = ({
                     }}>
                     {username || StringConstant.feedDeletedUserName}
                   </Text>
-                  {!isSelf && (
-                    <React.Fragment>
-                      <View style={styles.point} />
-                      <TouchableOpacity onPress={() => onPressFollUnFoll(isFollow)}>
-                        <Text style={isFollow ? styles.textFollowing : styles.textFollow}>
-                          {isFollow ? 'Following' : 'Follow'}
-                        </Text>
-                      </TouchableOpacity>
-                    </React.Fragment>
-                  )}
+                  {disabledFollow
+                    ? null
+                    : !isSelf && (
+                        <React.Fragment>
+                          <View style={styles.point} />
+                          <TouchableOpacity onPress={() => onPressFollUnFoll(isFollow)}>
+                            <Text style={isFollow ? styles.textFollowing : styles.textFollow}>
+                              {isFollow ? 'Following' : 'Follow'}
+                            </Text>
+                          </TouchableOpacity>
+                        </React.Fragment>
+                      )}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to my previous comment, the conditional rendering here is a good approach but make sure the `disabledFollow` prop is always passed to avoid unexpected behavior.



<!-- This is an auto-generated comment by OSS CodeRabbit -->